### PR TITLE
Fix colour and opacity of icons shown over main content

### PR DESCRIPTION
**Pull request by danxuliu**
This pull request fixes the colour and the opacity of the icons shown on the content (the app navigation toggle, the sidebar trigger, the fullscreen button and the media buttons).

@nextcloud/designers Note that each group of icons uses its own opacity: the app navigation toggle (which comes from the server) uses 0.6, the sidebar trigger and the fullscreen button use 0.5, and the media buttons use 0.7. I have respected the opacity that was being used for them because I was not sure if those different values were on purpose due to their different function, or it is just something that should be fixed (but, if that is the case, it can wait to a different pull request ;-) ).

Before, no call:
content-icons-no-call-before

After, no call:
content-icons-no-call-after

Before, call:
content-icons-call-before

After, call:
content-icons-call-after

### CODE REVIEW

File: css/style.scss [hunk 1]
```diff
@@ -449,27 +449,63 @@ video {
...
-		&:focus {
-			opacity: 1;
-		}
 	}
 }
 
+/* Use white icon for the app navigation toggle when in a call */
+#app-content.incall .icon-menu,
+#app-content.screensharing .icon-menu {
+	@include icon-color('menu', 'actions', $color-white, 1, true);
```
Comment (skjnldsv) [inline css/style.scss]: nice!

File: css/style.scss [hunk 2]
```diff
@@ -746,9 +799,7 @@ video {
 	top: 0;
 }
 
-.icon-menu-people {
-	background-image: url('../img/menu-people.svg?v=1');
-}
+@include icon-black-white('menu-people', 'spreed', 1);
```
Comment (skjnldsv) [inline css/style.scss]: :+1:

**Comment by nickvergessen** [comment]
On the first change of the camera, the white icons are not loaded and it takes a while until it shows up again, can we preload them somehow?

**Comment by skjnldsv** [comment]
@nickvergessen It only depends on the bandwith :/
Once cached it should be ok

**Pull request merged**